Onboarding/menu-personalised.md

```diff
@@ -0,0 +1,673 @@
+      query: [
+        ["slug", "mnch_onboarding_data_preferences_confirmation"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+
+  loading_message =
+    substitute(
+      message.message,
+      "{Text only/ Text & images/ Text, images, audio & video}",
+      "@data_preference"
+    )
+
+  button_labels = map(message.buttons, & &1.value.title)
+end
+
+card DisplayDataPreferencesConfirmation, then: DisplayDataPreferencesConfirmationError do
+  buttons(PersonalisedMenu: "@button_labels[0]") do
+    text("@loading_message")
+  end
+end
+
+card DisplayDataPreferencesConfirmationError, then: DisplayDataPreferencesConfirmationError do
+  buttons(PersonalisedMenu: "@button_labels[0]") do
+    text("@button_error_text")
+  end
+end
+
+```
+
+## Help Centre
+
+```stack
+card HelpCentre do
+  log("Help Centre")
+  run_stack("ea366b74-df7b-41ed-a479-7d501435d38e")
+end
+
+```
+
+## Your Profile
+
+```stack
+card YourProfile do
+  log("Your Progile")
+  run_stack("1f551cbc-db57-41d3-b5ee-dc6c77b3030b")
+end
+
+```
+
+## Take A Tour
+
+```stack
+card GoToTakeATour do
+  log("Take a tour")
+  run_stack("4288d6a9-23c9-4fc6-95b7-c675a6254ea5")
+end
+
+```
+
+## About and Privacy Policy
+
+```stack
+card AboutPrivacy, then: DisplayAboutPrivacy do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_about_privacy"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+end
+
+card DisplayAboutPrivacy, then: DisplayAboutPrivacyError do
+  doc_id = message.document
+
+  doc_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/documents/@doc_id/",
+      headers: [
+        ["Authorization", "Token @config.items.contentrepo_token"]
+      ]
+    )
+
+  document("@doc_data.body.meta.download_url")
+
+  buttons(MainMenu: "@button_labels[0]") do
+    text("@message.message")
+  end
+end
+
+card DisplayAboutPrivacyError, then: DisplayAboutPrivacyError do
+  buttons(MainMenu: "@button_labels[0]") do
+    text("@button_error_text")
+  end
+end
+
+```
+
+## TODO
+
+```stack
+card ArticleTopic01Secondary do
+  log("Placeholder")
```

Comment:
Maybe add a text("TODO: Article Topic") or something so that Meimoen can see, otherwise he's going to be confused.